pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -61,6 +61,7 @@ const (
 	attrRequestAudiences = "request.auth.audiences" // intended audience(s) for this authentication information.
 	attrRequestPresenter = "request.auth.presenter" // authorized presenter of the credential.
 	attrRequestClaims    = "request.auth.claims"    // claim name is surrounded by brackets, e.g. "request.auth.claims[iss]".
+	attrRequestGroups    = "groups"                 // groups claim, it is an alias to request.auth.claims[groups].
```

Comment:
Is it ok to add a new user facing attribute that's not covered in https://istio.io/docs/reference/config/policy-and-telemetry/attribute-vocabulary/?